tests/commands/prop.rb

```diff
@@ -0,0 +1,32 @@
+require 'minitest/autorun'
+require_relative '../../commands/prop'
+
+class PropCommandTest < Minitest::Test
+  def test_no_arguments
+    expected_output = <<~EOT
+      Explain the purpose of various package boolean properties.
+      Usage: crew prop <property>
+      Available properties: conflicts_ok, git_fetchtags, is_fake, is_musl, is_static, no_compile_needed, no_compress, no_env_options, no_fhs, no_links, no_patchelf, no_shrink, no_source_build, no_strip, no_lto, no_upstream_update, no_zstd, patchelf, print_source_bashrc, run_tests
```

Comment:
I'm wondering if it makes sense to calculate based on what exists add the list of properties to a variable or constant and then reuse that variable here instead?  I only say that because it's being used in 4 different places now.

Reply:
I looked into that-- the code for doing it is pretty ugly, and it also has the problem that we actually don't have documentation for a lot of those properties :sweat_smile:

It'd also mean we need to either duplicate the code, or add more dependencies, or pass around another argument.

I agree the current situation is not ideal, but when we add a new property we already have to manually add it to a list in `lib/package.rb`, a few more lists isn't too big of a deal.

Reply:
Remember the DRY method.  Don't Repeat Yourself.  It's a cardinal sin for developers.

Reply:
Added benefit: New properties are added to the list automagically.

Reply:
Probably not amazing if we add properties to the list independent of writing documentation-- I can see a scenario in which someone forgets to add documentation for a property but `crew` thinks we have documentation (In fact, this is the case right now with `git_clone_deep`, `arch_flags_override` and `gnome` (off the top of my head), so its not an imaginary concern.

(right now this PR doesn't have that issue-- it only says it has documentation for the properties we have documentation for-- master has this issue, and implementing things like this would cause the issue)

I think it probably makes sense for people to be adding to this list manually, at the same time they're adding the documentation for the property.

Reply:
The fact that you have split out these files is making things more complicated simply by abstraction and it would be very easy to miss something new.  I don't think making this many manual changes repeatedly is good for anyone's sanity.

Reply:
It's one identical change that has to be done four times, whenever someone adds a new property (which has been happening once every month recently, and that's a faster pace than usual). A find and replace would get the job done. 

If you just really do not want to have to make these changes whenever you add a property, I'm happy to write a script that you can use locally to avoid doing so, or just make the changes for you myself whenever you add a property.

My goal here is not to make things more difficult for you, and I'm happy with any compromise that preserves code simplicity and reduces inter-dependencies.

Reply:
> It's one identical change that has to be done four times,

Please figure out a way to do this only once. I agree with @uberhacker that making an identical change four times is problematic because it can be missed. 

Figure out a way to automate the change automatically. I don't care it it goes in a config file or whatever. `crew` should be able to figure out the rest on its own.

Reply:
By design, the tests ensure that if the change is made in `commands/prop.rb` they will fail unless they are also updated, so it can't be missed.

Personally, I think making an identical change 4 times is not particularly difficult, especially when it does not occur very often.

Reply:
But the most important thing here is that, again, I do not think it is a good idea to make these changes automatically. 

That introduces the possibility of `crew help prop` reporting that it has documentation for properties that it currently does not. 

In master, we generate this list of properties automatically. And in master, this is wrong. Multiple properties are automatically added to the list despite not having documentation available. 

This PR fixes that bug-- I do not want to re-introduce it.